css/splash.css

```diff
@@ -0,0 +1,106 @@
+.splash {
+    position: relative;
+}
+
+.splash img {
+    display: block;
+    width: 600px;
+    margin: 100px auto;
+}
+
+.splash .instructionsBtn {
+    display: block;
+    width: 600px;
+    height: 50px;
+    margin: 50px auto;
+    text-align: center;
+    font-size: 2rem;
+    font-family: 'Press Start 2P', cursive;
+    border-radius: 5px;
+    color: #ffffff;
+    text-shadow: 2px 2px #000000, 2px -2px #000000, -2px -2px #000000, -2px 2px #000000;
+    background-color: #dddddd;
+}
+
+.splash .instructionsBtn:hover {
+   cursor: pointer;
+   background-color: #dddddd;
```

Comment:
unused

Reply:
this is being used to colour the instructions button background to grey

Reply:
as discussed in person, this is a duplication of line 21

Reply:
done